src/chat.lisp

```diff
@@ -1,5 +1,10 @@
 (defpackage #:cl-telegram-bot/chat
   (:use #:cl)
+  (:import-from #:closer-mop
+                #:class-slots
+                #:slot-definition-initargs)
+  (:import-from #:cffi
+                #:translate-underscore-separated-name)
```

Comment:
I don't see where you are using this function. It is better to remove this unnecessary dependency.